native/src/components/Header.tsx

```diff
@@ -185,13 +192,22 @@ const Header = ({
     renderItem(HeaderButtonTitle.Language, 'language', showItems, goToLanguageChange),
   ]
 
+  const unsupportedLanguagesForTts = ['fa']
+  const ttsEnabled =
+    content &&
+    (buildConfig().appName === 'IntegreatTestCms' || buildConfig().appName === 'Integreat') &&
+    !unsupportedLanguagesForTts.includes(languageCode)
+      ? renderOverflowItem(t(`${HeaderButtonTitle.ReadAloud}`), onRead)
```

Comment:
just use here
```js
()=>setVisible(true)
```
and remove onRead, since its not reused somewhere and unneeded code